Submitted By: Wayne Blaszczyk
Date: 2009-10-27
Initial Package Version: 2.28.0
Origin: Upstream (http://bugzilla-attachments.gnome.org/attachment.cgi?id=144952)
Upstream Status: Unknown
Description: Fixes stickynotes crashing problem.

diff -Naur a/stickynotes/stickynotes.ui b/stickynotes/stickynotes.ui
--- a/stickynotes/stickynotes.ui	2009-09-12 12:54:54.000000000 +1000
+++ b/stickynotes/stickynotes.ui	2009-10-27 19:44:01.000000000 +1100
@@ -474,7 +474,7 @@
               </packing>
             </child>
             <child>
-              <object class="GtkVBox" id="vbox1">
+              <object class="GtkVBox" id="vbox3">
                 <property name="visible">True</property>
                 <property name="orientation">vertical</property>
                 <child>
@@ -494,7 +494,7 @@
                   </packing>
                 </child>
                 <child>
-                  <object class="GtkLabel" id="label1">
+                  <object class="GtkLabel" id="label2">
                     <property name="visible">True</property>
                     <property name="xalign">0</property>
                     <property name="label" translatable="yes">This cannot be undone.</property>
